SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3424,7 +3431,7 @@ void USpatialNetDriver::TryFinishStartup()
 	}
 	else
 	{
-		if (bMapLoaded)
+		if (ClientStartupHandler->TryFinishStartup())
```

Comment:
Should probably have a validity check

Reply:
Sad yes - this really has no business being `nullptr` on clients